docs/build.md

```diff
@@ -82,13 +82,21 @@ Distribution packages are created by exec `grunt build` on Linux platform (e.g.
 
 After installing the supported version of `node` and `npm`, install build dependency packages.
 
-Ubuntu/Debian:
+**Ubuntu/Debian:**
+
+```
+$ npm i -g grunt-electron-installer-debian
```

Comment:
In the [discussion](https://github.com/BoostIO/Boostnote/issues/3278#issuecomment-541713997) on issue #3278, you said:
> You didn't really need to install every packages on your global filesystem, it'll bloat your global node_modules/, and isn't recommended at all.

I agree with this point, and would like to see this documentation change to reflect your previous statement. Could we use the `yarn add` command you suggested in the same comment?
```
yarn add --dev grunt-electron-installer-debian grunt-electron-installer-debian
```

Reply:
I actually forgot we use yarn, not npm :sweat_smile:

Reply:
Resolved on 286ebd83b13b65a65e5da9d6f7d7797171b0348b.